backend/e4h-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -2618,9 +2991,59 @@ def find_col(partial):
+        custom_total_system_capacity_col = find_col("Custom Total System Capacity")
+
+        # MDMS schema for facilityType/systemType/solarSolutionDesignType/totalSystemCapacity
+        # code lookups (see build_field_plan_facility_additional_details) - falls back to raw
+        # Excel labels if the schema can't be fetched, same graceful-degradation pattern used
+        # elsewhere in this endpoint for other external service calls.
+        field_plan_facility_schema = []
+        try:
+            field_plan_facility_schema = mdms_client.get_column_definitions_with_metadata(
+                request_info, 'data-ingestion.FieldPlanFacilityIngestionSchema'
+            )
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)